### ISSUE TITLE
Texture from video tag leaks memory

### ISSUE THREAD

**Issue by geduldig** [mrdoob/three.js#9440]
##### Description of the problem 

I render a video file to a texture as follows: 
  - Create a texture from a video tag.
  - Set the video loop property to true.
  - Update the texture in requestanimationframe.
I expect memory usage to be stable. Instead the memory usage slowly increases.

The problem is easily repeatable by doing the following:
  - Open http://stemkoski.github.io/Three.js/Video.html in Chrome.
  - Wait for the video to play once.
  - In devtools console, execute these commands:
      - > video.loop=true
      - > video.play()
  - Open Chrome's Task Manager, and keep your eye on the Memory column for the GPU Process task.

Memory usage increases several megabytes in about 10 minutes. I let this run for a day and the memory usage was still increasing.

##### Three.js version

- [x] Dev
- [x] r79

##### Browser

- [x] Chrome
- [x] Firefox
- [x] Safari

##### OS

- [x] Windows
- [x] OS X

##### Hardware Requirements (graphics card, VR Device, ...)

Nvidia Quadro FX 4800

**Comment by danrossi** [comment]
You are drawing to both webgl and canvas context ? 

```
videoImageContext.drawImage( video, 0, 0 );
```

Disable that ?

**Comment by geduldig** [comment]
Why would you disable this? Then the context would not get rendered to. This is the whole point of  rendering video to a texture.

**Comment by mrdoob** [comment]
Does this happen using [VideoTexture](https://github.com/mrdoob/three.js/blob/master/src/textures/VideoTexture.js) too?

**Comment by mrdumb** [comment]
Here is a much more complex example:
http://necromanthus.com/Test/html5/testA.html 
Using the latest Firefox (47.0.1) I get almost the same memory usage after 5 or 50 minutes.

**Comment by geduldig** [comment]
@mrdoob: Like you did, I observed running your example (and mine) in Firefox did not produce a memory leak. It also appears there have been other issues reported of memory leaks in Chrome's GPU Process, which leads me to believe I found another instance of this. So, I filed a new issue with Chromium here:
https://bugs.chromium.org/p/chromium/issues/detail?id=470234

@mrdumb: Thanks for posting your example. I didn't see a looping video inside, unless one manually replays the Katy Perry video, which makes it difficult to use as a test case. Nice example though!
